pages/academicIcon.tsx

```diff
@@ -1,7 +1,7 @@
 const academicIcon: React.FunctionComponent = () => (
```

Comment:
same as here, all the icons should be components since they'll be elements of a page and shouldn't take up an entire page